From b9bd56a756d42cdc7fc50ff45fe7ab7264421768 Mon Sep 17 00:00:00 2001
From: Nick Zivkovic <nick.zivkovic@joyent.com>
Date: Mon, 29 Aug 2016 23:48:52 +0000
Subject: [PATCH] MANATEE-312 manatee should not set snapdir to visible
 Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved by: Dave Pacheco
 <dap@joyent.com>

---
 tools/mkdevsitters | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/mkdevsitters b/tools/mkdevsitters
index a6dc936..92b23aa 100755
--- a/tools/mkdevsitters
+++ b/tools/mkdevsitters
@@ -33,7 +33,7 @@ function mksitter
 
 	echo -n "peer $which: creating ZFS dataset ... "
 	if ! zfs create -o mountpoint=$mountpoint -o canmount=noauto \
-	    -o snapdir=visible $dataset; then
+	    $dataset; then
 		fail "failed to create dataset \"$dataset\""
 	fi
 
-- 
2.21.0

